xsd/netex_part_3/part3_fares/netex_fareSeries_version.xsd

```diff
@@ -328,5 +335,72 @@ Rail transport, Roads and Road transport
 			<xsd:element ref="SeriesConstraintRef" minOccurs="0"/>
 		</xsd:sequence>
 	</xsd:group>
+	<!-- == ZONE IN SERIES================================ -->
+	<xsd:complexType name="zonesInSeries_RelStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for a list of ZONEs IN SERIES.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="strictContainmentAggregationStructure">
+				<xsd:sequence>
+					<xsd:element name="ZoneInSeries" type="ZoneInSeries_VersionedChildStructure" maxOccurs="unbounded">
+						<xsd:annotation>
+							<xsd:documentation>An orderd sequence of ZONEs IN SEQUENCE constrainign the permitted routing.</xsd:documentation>
+						</xsd:annotation>
+					</xsd:element>
+				</xsd:sequence>
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
+	<xsd:element name="ZoneInSeries" substitutionGroup="VersionedChild">
+		<xsd:annotation>
+			<xsd:documentation>Use of a ZONE in sequence within a SERIES CONSTRAINT, for example to indicate that a series must route through a given sequence of TARIFF ZONEs, +v2.0
+</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexType>
+			<xsd:complexContent>
+				<xsd:restriction base="ZoneInSeries_VersionedChildStructure">
+					<xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="EntityInVersionGroup" minOccurs="0"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="VersionedChildGroup"/>
+						</xsd:sequence>
+						<xsd:sequence>
+							<xsd:group ref="ZoneInSeriesGroup"/>
+						</xsd:sequence>
+					</xsd:sequence>
+					<xsd:attribute name="id" type="ZoneInSeriesIdType"/>
+				</xsd:restriction>
+			</xsd:complexContent>
+		</xsd:complexType>
+	</xsd:element>
+	<xsd:complexType name="ZoneInSeries_VersionedChildStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for a ZONE IN SERIES.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexContent>
+			<xsd:extension base="VersionedChildStructure">
+				<xsd:sequence>
+					<xsd:group ref="ZoneInSeriesGroup"/>
+				</xsd:sequence>
+				<xsd:attribute name="order" type="xsd:integer"/>
```

Comment:
@Aurige this should be a positiveInteger?

Reply:
that's right, this is the new rule ...